rs/rosetta-api/icrc1/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::disallowed_types)]
```

Comment:
Instead of allowing all disallowed types in the _entire file_, couldwe apply this allow-rule only to the line(s) where it actually applies? This way we would catch disallowed types in files where other disallowed types are already present. If there are multiple/too many occurrences, maybe an alias type like the one used in line 16 (`use tokio::sync::Mutex as AsyncMutex;`) could help.

Reply:
I just didn't want to track it down because as you say maybe they are multiple uses of the disallowed types. I would live it this to the code owners to handle those.